src/ElectricPokemon.java

```diff
@@ -0,0 +1,90 @@
+import java.util.Arrays;
+import java.util.List;
+
+public class ElectricPokemon extends Pokemon {
+    private String specialAttack;
+    private String strongAgainst;
+
+    public ElectricPokemon(String name, int hp, int level, String food, String sound) {
+        super(name, hp, level, food ,sound, "electric");
+
+    }
+
+    public String getSpecialAttack() {
+        return specialAttack;
+    }
+
+    public String getStrongAgainst() {
+        return strongAgainst;
+    }
+
+    public List<String> getAttacks() {
+        return Arrays.asList("thunderPunch", "electroBall", "thunder", "voltTackle");
+    }
+
+
+    public void thunderPunch(Pokemon attacker, Pokemon enemy) {
+        System.out.println(attacker.getName() + " attacks " + enemy.getName() + " with thunderPunch!");
+        int damage;
+        if (enemy.getType().equalsIgnoreCase("water")) {
```

Comment:
Het is inderdaad goed om je zelf te beschermen tegen foutief hoofdletter gebruik, maar eigenlijk zou dat niet voor moeten komen. Je hebt de code immers zelf geschreven en dit is geen user input. 
Wel een good practice. 
Idealiter zou je hier zelfs een enum kunnen gebruiken.

Reply:
Een switch statement was op deze plek misschien handiger geweest.